config/crd/bases/starlingx.windriver.com_hostprofiles.yaml

```diff
@@ -580,9 +580,9 @@ spec:
                 description: Location defines the physical location of the host in
                   the data centre.
                 type: string
-              maxCPUFrequency:
-                description: MaxCPUFrequency defines the maximum limit of the CPU
-                  frequency set on the host.
+              max_cpu_mhz_configured:
```

Comment:
Name mismatch with the definition in the type.go

Reply:
Renamed the field to maxCPUMhzConfigured

Reply:
Since it is declared as max_cpu_mhz_configured [here ](https://github.com/Wind-River/gophercloud/pull/23/files#diff-17717160b479478faeca74dc8ec86bf1eb72eeb1801b047d6ccf301e528fbbc4R253), when we run deploy tool, it generates with this name automatically